lib/DriverTool/swift_symbolgraph_extract_main.cpp

```diff
@@ -171,7 +171,7 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args,
       ParsedArgs.hasArg(OPT_skip_inherited_docs),
       ParsedArgs.hasArg(OPT_include_spi_symbols),
       /*IncludeClangDocs=*/false,
-  };
+      ParsedArgs.hasArg(OPT_emit_extension_block_symbols)};
```

Comment:
```suggestion
      ParsedArgs.hasArg(OPT_emit_extension_block_symbols)
    };
```

Reply:
Interestingly, you need to use trailing comma syntax, otherwise clang-format will revert your suggested change to what  I committed.